en/docs/apis/restapis/idp.yaml

```diff
@@ -505,6 +505,184 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
+  '/identity-providers/{identity-provider-id}/export':
+    get:
+      tags:
+        - Identity Providers
+      summary: |
+        Export identity provider in XML, YAML, or JSON file formats
```

Comment:
do we export the IdP or is it the IdP details?

Reply:
We are exporting the IDP object. Used `Export identity provider` to be consistent with other API descriptions.